src/methods/democracy/openProxy.ts

```diff
@@ -5,26 +5,26 @@ import {
   UnsignedTransaction
 } from '../../util';
 
-export interface DemocracyRemoveProxyArgs extends Args {
+export interface DemocracyOpenProxyArgs extends Args {
   /**
    * The address of the proxy to remove, SS-58 encoded.
    */
-  proxy: string;
+  target: string;
 }
 
 /**
- * Clear the proxy. Called by the stash.
+ * Become a proxy. This must be called prior to a later `activate_proxy`.
```

Comment:
Not sure if you normally keep the rust or ts style on these.
```suggestion
 * Become a proxy. This must be called prior to a later `activateProxy`.
```

Reply:
let's keep the TS style since this is a TS lib. (i actually just copy-pasted, that's why there's a _)